include/deal.II/dofs/dof_tools.h

```diff
@@ -1248,12 +1248,23 @@ namespace DoFTools
    *
    * The size of @p selected_dofs shall equal <tt>dof_handler.n_dofs()</tt>.
    * Previous contents of this array or overwritten.
+   *
+   * In case of a parallel::shared::Triangualtion or a
+   * parallel::distributed::Triangulation only locally relevant dofs are considered.
```

Comment:
Please add a sentence like this: "Note that the vector returned through the second argument still has size `DoFHandler::n_dofs()`. Consequently, it can be very large for large parallel computations -- in fact, it may be too large to store on each processor. In that case, you may want to choose the variant of this function that returns an IndexSet object."